### PR TITLE
add ruta-zuke.html

### PR DESCRIPTION
**Pull request by rutatulis13**

### CODE REVIEW

**Review by MartynasKupsys — APPROVED** [review]
Gooood

**Review by kstulgys — APPROVED** [review]
Looks good, very clean

**Review by seinauskaite — APPROVED** [review]

**Review by mariuli — APPROVED** [review]

File: ruta-zuke.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h1>Hi, I am Ruta.</h1>
+      <h2>Amateur photographer.</h2>
```
Comment (eciras) [inline ruta-zuke.html:12]: This should be <p> tag :)

**Review by eciras — APPROVED** [review]

**Review by MarcinkEve — APPROVED** [review]

**Review by kstulgys — CHANGES_REQUESTED** [review]
Please delete .DS_Store file

**Review by suiraDN — APPROVED** [review]

**Review by MrStonkus — APPROVED** [review]

**Comment by MrStonkus** [comment]
Well done.

File: ruta-zuke.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ruta Zuke</title>
+  </head>
+  <body>
+    <header>
+      <h1>Hi, I am Ruta.</h1>
+      <p>Amateur photographer.</p>
```
Comment (kstulgys) [inline ruta-zuke.html:12]: Weel done

**Review by kstulgys — APPROVED** [review]

**Pull request merged**